pulsar-broker/src/test/java/org/apache/pulsar/client/impl/TransactionEndToEndTest.java

```diff
@@ -249,6 +246,63 @@ public void produceAbortTest() throws Exception {
         log.info("finished test partitionAbortTest");
     }
 
+    @Test(dataProvider="enableBatch")
+    private void testAckWithTransactionReduceUnAckMessageCount(boolean enableBatch) throws Exception {
```

Comment:
The test should enable batch index ack for the broker and the consumer?

Reply:
TransactionTestBase has enable batch index ack